server/core/src/configuration_hook_manager.cpp

```diff
@@ -0,0 +1,22 @@
+#include "irods/configuration_hook_manager.hpp"
+#include <fmt/format.h>
+#include "irods/irods_logger.hpp"
+namespace irods::experimental::hooks{
+  configuration_hook_manager::configuration_hook_manager(){}
```

Comment:
i think we're four spaces for indentation.... does the clang formatter pick this up?

Reply:
Yes. It's captured in the clang-format github action output.

Reply:
I didn't notice `git clang-format` until I was a few commits in.